spec/system/admin/adding_to_an_order_spec.rb

```diff
@@ -187,8 +187,10 @@
 
   describe "adding a product from another facility", :js, feature_setting: { cross_core_projects: true } do
     let(:facility2) { create(:setup_facility) }
-    let(:product) { create(:setup_item, :with_facility_account) }
-    let!(:product2) { create(:setup_item, :with_facility_account, facility: facility2) }
+    let(:product) { create(:setup_item, :with_facility_account, cross_core_ordering_available: true) }
+    let!(:product2) { create(:setup_item, :with_facility_account, facility: facility2, cross_core_ordering_available: true) }
+    let!(:product3) { create(:setup_item, :with_facility_account, facility:, cross_core_ordering_available: false) }
+    let!(:product4) { create(:setup_item, :with_facility_account, facility: facility2, cross_core_ordering_available: false) }
```

Comment:
Can you give these descriptive names like cross_core_product, cross_core_original_facility, etc?